hardware.inc

```diff
@@ -731,19 +739,29 @@ def rRAMG equ $0000
 def RAMG_SRAM_DISABLE equ $00
 def RAMG_SRAM_ENABLE  equ $0A ; some MBCs accept any value whose low nybble is $A
 
-; -- ROMB0 ($2000-$3FFF) ------------------------------------------------------
-; ROM bank number (low 8 bits when applicable) [wo]
-def rROMB0 equ $2000
+def RAMG_SRAM_IR equ $0E ; (HuC1 only) switch SRAM to IR mode
 
-; -- ROMB1 ($3000-$3FFF) ------------------------------------------------------
-; (MBC5 only) ROM bank number high bit (bit 8) [wo]
-def rROMB1 equ $3000
+; (HuC-3 only) switch SRAM to map cartridge RAM, RTC, or IR
+def RAMG_CART_RAM_RO   equ $00 ; select cartridge RAM [ro]
+def RAMG_CART_RAM      equ $0A ; select cartridge RAM [r/w]
+def RAMG_RTC_IN        equ $0B ; select RTC command/argument [wo]
+    def RAMG_RTC_IN_CMD equ %0_111_0000 ; command
+    def RAMG_RTC_IN_ARG equ %0000_1111  ; argument
```

Comment:
This is only an idea, but what about something like this:

```asm
    def RAMG_RTC_IN_CMD equ %0_111_0000 ; command
    def RAMG_RTC_IN_ARG equ %0_000_1111 ; argument
```
keeping the groupings vertically aligned.

Reply:
Yeah, good idea!